### ISSUE TITLE
UserWarning: Overwriting wide_resnet101_2 in registry with braincog.model_zoo.resnet.wide_resnet101_2. This is because the name being registered conflicts with an existing name. Please check if this is not expected.

### ISSUE THREAD

**Issue by tankhang2004** [BrainCog-X/Brain-Cog#186]
# 'from braincog.base.node import LIFNode' I ran this import then this log appears:

"/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:461: UserWarning: Overwriting resnet18 in registry with braincog.model_zoo.resnet.resnet18. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnet18(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:474: UserWarning: Overwriting resnet34 in registry with braincog.model_zoo.resnet.resnet34. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnet34(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:486: UserWarning: Overwriting resnet50 in registry with braincog.model_zoo.resnet.resnet50. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnet50(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:491: UserWarning: Overwriting resnet101 in registry with braincog.model_zoo.resnet.resnet101. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnet101(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:497: UserWarning: Overwriting resnet152 in registry with braincog.model_zoo.resnet.resnet152. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnet152(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:503: UserWarning: Overwriting resnext50_32x4d in registry with braincog.model_zoo.resnet.resnext50_32x4d. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnext50_32x4d(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:511: UserWarning: Overwriting resnext101_32x8d in registry with braincog.model_zoo.resnet.resnext101_32x8d. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def resnext101_32x8d(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:519: UserWarning: Overwriting wide_resnet50_2 in registry with braincog.model_zoo.resnet.wide_resnet50_2. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def wide_resnet50_2(pretrained=False, **kwargs):
/Users/khang/Documents/F-Brain-Cog/braincog/model_zoo/resnet.py:526: UserWarning: Overwriting wide_resnet101_2 in registry with braincog.model_zoo.resnet.wide_resnet101_2. This is because the name being registered conflicts with an existing name. Please check if this is not expected.
  def wide_resnet101_2(pretrained=False, **kwargs):"

# This is all the packages existing in my environment conda 
"conda list
# packages in environment at /Users/khang/miniconda3/envs/braincog:
#
# Name                    Version                   Build  Channel
absl-py                   2.1.0                    pypi_0    pypi
audioread                 3.0.1                    pypi_0    pypi
braincog                  0.2.7.19                  dev_0    <develop>
ca-certificates           2024.3.11            hca03da5_0  
cachetools                5.3.3                    pypi_0    pypi
certifi                   2024.2.2                 pypi_0    pypi
cffi                      1.16.0                   pypi_0    pypi
charset-normalizer        3.3.2                    pypi_0    pypi
contourpy                 1.1.1                    pypi_0    pypi
cycler                    0.12.1                   pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
deprecated                1.2.14                   pypi_0    pypi
dv                        1.0.12                   pypi_0    pypi
einops                    0.7.0                    pypi_0    pypi
expelliarmus              1.1.12                   pypi_0    pypi
filelock                  3.13.3                   pypi_0    pypi
flatbuffers               24.3.25                  pypi_0    pypi
fonttools                 4.50.0                   pypi_0    pypi
fsspec                    2024.3.1                 pypi_0    pypi
google-auth               2.29.0                   pypi_0    pypi
google-auth-oauthlib      1.0.0                    pypi_0    pypi
grpcio                    1.62.1                   pypi_0    pypi
h5py                      3.10.0                   pypi_0    pypi
huggingface-hub           0.22.2                   pypi_0    pypi
idna                      3.6                      pypi_0    pypi
importlib-metadata        7.1.0                    pypi_0    pypi
importlib-resources       6.4.0                    pypi_0    pypi
importrosbag              1.0.4                    pypi_0    pypi
jinja2                    3.1.3                    pypi_0    pypi
joblib                    1.3.2                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
lazy-loader               0.3                      pypi_0    pypi
libcxx                    14.0.6               h848a8c0_0  
libffi                    3.4.4                hca03da5_0  
librosa                   0.10.1                   pypi_0    pypi
llvmlite                  0.41.1                   pypi_0    pypi
lz4                       4.3.3                    pypi_0    pypi
markdown                  3.6                      pypi_0    pypi
markupsafe                2.1.5                    pypi_0    pypi
matplotlib                3.7.5                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
msgpack                   1.0.8                    pypi_0    pypi
ncurses                   6.4                  h313beb8_0  
networkx                  3.1                      pypi_0    pypi
numba                     0.58.1                   pypi_0    pypi
numpy                     1.24.4                   pypi_0    pypi
oauthlib                  3.2.2                    pypi_0    pypi
openssl                   3.0.13               h1a28f6b_0  
packaging                 24.0                     pypi_0    pypi
pandas                    2.0.3                    pypi_0    pypi
pbr                       6.0.0                    pypi_0    pypi
pillow                    10.3.0                   pypi_0    pypi
pip                       23.3.1           py38hca03da5_0  
platformdirs              4.2.0                    pypi_0    pypi
pooch                     1.8.1                    pypi_0    pypi
protobuf                  5.26.1                   pypi_0    pypi
pyasn1                    0.6.0                    pypi_0    pypi
pyasn1-modules            0.4.0                    pypi_0    pypi
pycparser                 2.22                     pypi_0    pypi
pygame                    2.5.2                    pypi_0    pypi
pyparsing                 3.1.2                    pypi_0    pypi
python                    3.8.19               hb885b13_0  
python-dateutil           2.9.0.post0              pypi_0    pypi
pytz                      2024.1                   pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
readline                  8.2                  h1a28f6b_0  
requests                  2.31.0                   pypi_0    pypi
requests-oauthlib         2.0.0                    pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
safetensors               0.4.2                    pypi_0    pypi
scikit-learn              1.3.2                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
seaborn                   0.13.2                   pypi_0    pypi
setuptools                68.2.2           py38hca03da5_0  
six                       1.16.0                   pypi_0    pypi
soundfile                 0.12.1                   pypi_0    pypi
soxr                      0.3.7                    pypi_0    pypi
sqlite                    3.41.2               h80987f9_0  
sympy                     1.12                     pypi_0    pypi
tensorboard               2.14.0                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
thop                      0.1.1-2209072238          pypi_0    pypi
threadpoolctl             3.4.0                    pypi_0    pypi
timm                      0.9.16                   pypi_0    pypi
tk                        8.6.12               hb8d0fd4_0  
tonic                     1.4.3                    pypi_0    pypi
torch                     2.2.2                    pypi_0    pypi
torchaudio                2.2.2                    pypi_0    pypi
torchvision               0.17.2                   pypi_0    pypi
tqdm                      4.66.2                   pypi_0    pypi
typing-extensions         4.10.0                   pypi_0    pypi
tzdata                    2024.1                   pypi_0    pypi
urllib3                   2.2.1                    pypi_0    pypi
werkzeug                  3.0.2                    pypi_0    pypi
wheel                     0.41.2           py38hca03da5_0  
wrapt                     1.16.0                   pypi_0    pypi
xz                        5.4.6                h80987f9_0  
zipp                      3.18.1                   pypi_0    pypi
zlib                      1.2.13               h5a0b063_0  
zstd                      1.5.5.1                  pypi_0    pypi"

**I don't know what the log means so I wonder if there is something wrong.
Thank you in advance**
